### ISSUE TITLE
Feature request: Easier permanent colour change for wires

### ISSUE THREAD

**Issue by IAmMarcelJung** [FPGA-Research-Manchester/FABulator#6]
Right now it takes three clicks to change the colour of a wire: Right click for opening the dialog, left click on the dialog and another left click to select the colour. This could be reduced to two or even one click if a color was previously selected. One click: Press a key (I suggest ```ctrl``` or ```shift```) while selecting a wire. This would need the implementation of #3 . Two clicks: Right click on wire, then the dialog should show an option for either selecting a colour or just using the last used color. Of course also both options could be implemented.

**Comment by JakobTernes** [comment]
I agree it's a bit of a hassle right now.
I suggest an extra tab menu for appearance-related edits.
There, a color could be selected, such that simply clicking on a wire colors it with the selected color, possibly requiring an extra key as you suggested.
Other features could be added there too, like adding custom labels to the fabric. I think that could be helpful too.

As a sidenote: you can already color in 2 clicks by clicking on the right to the color-chooser. That will select the last selected color. But it is a bit tricky to use and not documented...

**Comment by IAmMarcelJung** [comment]
Yes nice, thanks for mentioning the existing two click option! But I guess this is can easily be overlooked. Maybe the dropdown menu should at least be aligned to the right side, so you don't have to move the mouse more for selecting a new colour than for just choosing the last colour (so through lazyness you could already find this option).
